pipelines/Protcon-pipeline/Protcon-pipeline.Rmd

```diff
@@ -0,0 +1,180 @@
+---
+title: "Protcon pipeline documentation"
+output: 
+  github_document:
+    toc: true             
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(knitr)
+knitr::opts_chunk$set(echo = TRUE)
+require("knitr")
+
+```
+
+## General description
+<div style="text-align: justify">Protected areas are the most widely applied conservation strategy in the world and can maintain an acceptable representation of biodiversity and its processes. However, connectivity is an essential part of an effective system of areas and is rarely taken into account. The ability of the landscape to facilitate or impede the movement of species between protected areas is crucial for processes such as dispersal and gene flow to circulate between patches and to maintain the stability and integrity of different ecosystems over time.
+
+This document explains the pipeline flow for the protected areas connectivity indicator, which refers to the percentage of protected areas that are well connected given a distance and consultation area determine by the user.This routine can be applied to other types of spatial units other than protected areas, such as habitat patches or nodes of natural cover.<div/>
+
+## Authorship
+
+<div style="text-align: justify">This pipeline uses products at the national or international level that spatially define the units or nodes with which the analysis is to be run. It is common that layers of the distribution of protected areas or patches of natural areas are used to analyze their connectivity within a given landscape. Likewise, the methodology for calculating the connectivity of protected areas that was used for this indicator and this pipeline is based on the *protcon* indicator proposed by [Saura et al.in 2017](https://www.sciencedirect. com/science/article/pii/S1470160X1630752X?via%3Dihub).
+
+This methodology is based on a minimum distance in which protected areas are considered to be well connected; of course, this distance may vary according to the objective for which connectivity is desired. Once the protected areas that are at a distance equal to or less than the defined distance have been identified, the percentage that these represent with respect to the total protected areas of the total area of analysis (protcon) is estimated. To extract the percentage of protected areas that are not well connected (protuncon), the protcon is subtracted from the total number of protected areas in the area of interest. The indicator also generates the percentage values of protected (protected) and unprotected (unprotected) area of interest.
+
+$$
+protcon=\frac{A_{pac}}{A_{tpa}}*100
+$$
+**Where:**
```

Comment:
leave a blank line after `$$` just in case

Reply:
done

Reply:
done